app/src/main/java/org/p2p/wallet/rpc/repository/account/RpcAccountRemoteRepository.kt

```diff
@@ -71,10 +115,10 @@ class RpcAccountRemoteRepository(private val api: RpcAccountApi) : RpcAccountRep
 
         val programIds = listOf(
             TokenProgram.PROGRAM_ID,
-            TokenProgram.PROGRAM_ID_TOKEN2022
```

Comment:
i liked previous name more

Reply:
Me too, but in official solana docs, they also call this TOKEN_2022_PROGRAM_ID, so I decided to standardize it, and of course we already have `ASSOCIATED_TOKEN_PROGRAM_ID` which also matches this naming: ${TYPE}_PROGRAM_ID